src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -72,8 +70,10 @@
           :url="item.path || ''"
           :isActive="!isGroupMarker"
           :class="{ bolded: isBold }"
+          :tabindex="isFocused ? '0' : '-1'"
+          :aria-describedby="`${ariaDescribedBy} ${usageLabel}`"
+          :aria-hidden="isRendered ? null : 'true'"
```

Comment:
We should probably keep the aria-hidden on the parent. This is a failsafe in case you try to focus something that is used to render content.

Reply:
Moved :) https://github.com/apple/swift-docc-render/pull/209/commits/fd0a2b94c3d4094ce4c33a2c1a514afd55e6bd89